requirements-dev.txt

```diff
@@ -34,4 +34,4 @@ pypandoc
 sphinx
 sphinx-gallery
 git+https://github.com/Holzhaus/sphinx-multiversion.git
-pydata-sphinx-theme>=0.5.0
\ No newline at end of file
+pydata-sphinx-theme==0.5.2
```

Comment:
ideally you'd fix this on `main`, in a separate PR, and then `cherry-pick` on the release branch.

Reply:
This specific fix is now in #740 , which I'll cherry pick to `release/v0.6.1`. Technically this part doesn't need to go into the release branch, since documentation is always built from `main` (with the different versions due to `sphinx-multiversion` doing its thing.

I also plan on opening an issue to update to `pydata-sphinx-theme` v0.6 (how dare they have the same version as our current release! ;-) ).